lib/kamal/cli/main.rb

```diff
@@ -1,5 +1,6 @@
 class Kamal::Cli::Main < Kamal::Cli::Base
   desc "setup", "Setup all accessories, push the env, and deploy app to servers"
+  option :skip_push, aliases: "-P", type: :boolean, default: false, desc: "Skip image build and push"
```

Comment:
Please add tests for the `skip_push` option similar to https://github.com/basecamp/kamal/blob/main/test/cli/main_test.rb#L40

Reply:
Looks like tests were added :slightly_smiling_face: 

Is there something that prevents us from having this wonderful feature in the release? :slightly_smiling_face: